schemas/org.notes.gschema.xml

```diff
@@ -91,7 +91,7 @@
 		</key>
 
 		<key name="editor-font" type="s">
-			<default>"Open Sans 12"</default>
+			<default>"Open Sans 10"</default>
```

Comment:
Why the change to 10?

Reply:
I thought it was more in line with how fonts on the system are sized, I can change it back if you don't like it/don't think it's good.